File: judge/java/src.java
package judge;
import java.util.*;
import java.lang.*;
import java.io.*;
import tests.*;

public class src {
int second_largest(int[] arr) {
    if(arr.length < 2) return 0;
    int second_max = arr[0], max_val = arr[0];
    for(int i = 1; i < arr.length; ++i) {
        if(arr[i] > max_val) {
            second_max = max_val;
            max_val = arr[i];
        } else if(arr[i] > second_max && arr[i] != max_val) {
            second_max = arr[i];
        }
    }
    return second_max == max_val ? 0 : second_max;
}

public static void main (String[] args) throws java.lang.Exception {

    src s = new src();

    long startTime = System.currentTimeMillis();

    for(int i = 0; i < second_largest_number.num_test; ++i) {
        int answer = s.second_largest(second_largest_number.in_0[i]);
        if(answer != second_largest_number.out[i]) {
            System.out.printf("%d / %d;", i+1, second_largest_number.num_test);
            String outs = common.to_string(second_largest_number.in_org_0[i]);
            System.out.print(outs + ";");
            System.out.print(common.to_string(answer) + ";");
            System.out.println(common.to_string(second_largest_number.out[i]));
            return;
        }
    }
    long estimatedTime = System.currentTimeMillis() - startTime;
    System.out.print("Accepted;");
    System.out.println(estimatedTime);
}
}

File: judge/python/src.py
import sys
import datetime
from common import *
from tests.second_largest_number import *

def second_largest(arr):
    if len(arr) < 2: return 0
    second_max, max_val = arr[0], arr[0]
    for i in range(1, len(arr)):
        if arr[i] > max_val:
            second_max = max_val
            max_val = arr[i]
        elif arr[i] > second_max and arr[i] != max_val:
            second_max = arr[i]
    if second_max == max_val: return 0
    else: return second_max


start_time = datetime.datetime.now()

for i in range(num_test):
    answer = second_largest(in_0[i])
    if answer != out[i] :
        out_str = ""
        out_str += str(i+1) + " / " + str(num_test) + ";"
        out_str += str(in_org_0[i])
        out_str += ";"
        out_str += str(answer)
        out_str += ";"
        out_str += str(out[i])
        print(out_str)
        sys.exit(0)
delta = datetime.datetime.now() - start_time
runtime = str(int(delta.total_seconds() * 1000))
print('Accepted;' + runtime)
